opentreemap/treemap/css/sass/partials/pages/_treedetails.scss

```diff
@@ -98,10 +98,14 @@
     border-radius: 9px;
 }
 
-// Needed because bootstrap breaks street view on firefox
-// by setting max-width: 100% on all <img> tags
-.street-view-small img {
-    max-width: none;
+.street-view-small {
+    height: 440px;
```

Comment:
Because it gave a nice 2 to 1 size ratio of the box.  I explored going larger but then you start getting black bars on the top and bottom.  We could go smaller...